.github/workflows/release.yml

```diff
@@ -191,8 +191,20 @@ jobs:
     - name: Check out code
       uses: actions/checkout@v4
 
-    - name: Docker
-      run: make docker
+    - name: Extract VERSION from Makefile
+      id: get_version
+      run: |
+        VERSION=$(make --no-print-directory get-version)
+        echo "version=$VERSION" >> $GITHUB_OUTPUT
+
+    - name: Build ARM64 and AMD64 Docker images
+      uses: docker/build-push-action@v5
```

Comment:
Can't we use an arm runner like `ubuntu-24.04-arm` to push architecture specific tags and combine them under a docker manifest? We do that internally a lot. Finding free arm runners (for OSS) is hard, but more are coming out. 

https://github.com/actions/partner-runner-images?tab=readme-ov-file#available-images